### PR TITLE
efl 1.18.1

### PR DESCRIPTION
**Pull request by jeanguyomarch**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

Since EFL 1.18.0, EFL, Evas-Generic-Loaders and Elementary have been merged.

### CODE REVIEW

**Comment by jeanguyomarch** [comment]
Hi,

I can see that the automatic checks that failed (https://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/7932/#showFailuresLink) are related to elementary and evas-generic-loaders. However, this patch move these two formulas to the boneyard, because since EFL 1.18, elementary and evas-generic-loaders have been merged into core efl. Therefore, there is only one formula for efl: `efl` itself.

So I'm not quite sure what to do next.
Thanks.

File: tap_migrations.json [hunk 1]
```diff
@@ -67,9 +67,11 @@
   "eet": "homebrew/boneyard",
   "eina": "homebrew/boneyard",
   "electric-fence": "homebrew/boneyard",
+  "elementary": "homebrew/boneyard",
```
Comment (MikeMcQuaid) [inline tap_migrations.json]: You need to delete the formula file too.

Comment (jeanguyomarch) [inline tap_migrations.json]: Thanks for the review. I will send a revised patch.

**Comment by dunn** [comment]
Please open PRs for the removed formula at https://github.com/Homebrew/homebrew-boneyard, thanks!

**Comment by jeanguyomarch** [comment]
I opened a PR to remove elementary and evas-generic-loaders in the boneyard: https://github.com/Homebrew/homebrew-boneyard/pull/185. Thanks.

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
For some reason it's trying to download the nonexistent bottle, instead of building from source.

**Comment by MikeMcQuaid** [comment]
@dunn `brew test-bot` bug. Requeued.

**Comment by dunn** [comment]
[daft punk voice] one more time

**Comment by dunn** [comment]
Finally merged as 49c83fd2363078fade96547520a8b6156db9810b, thanks!

**Pull request closed without merging**